libraries/AP_DDS/README.md

```diff
@@ -10,21 +10,21 @@ title: UDP Loopback
 ---
 graph LR
 
-  subgraph Linux Computer
+subgraph Linux Computer
 
-    subgraph Ardupilot SITL
-      veh[sim_vehicle.py] <--> xrceClient[EProsima Micro XRCE DDS Client]
-      xrceClient <--> port1[udp:2019]
-    end
+subgraph Ardupilot SITL
```

Comment:
Did you accidentally re-format the whole README? This indent is now messed up.

Reply:
I ran the astyle on the readme file..no worries..have reverted the change

Reply:
Thanks, yea `astyle` is only used for the C++ code.